intermittent-failure-tracker/init.sls

```diff
@@ -37,6 +40,12 @@ intermittent-failure-tracker:
     - group: servo
     - mode: 644
 
+/home/servo/intermittent-failure-tracker/intermittent_errors.json:
```

Comment:
Why do we need to create this file ourselves? Seems like something the package should create on-demand.

Reply:
The current working directory is not owned by the user that is running the server, so it can't.